Apps/Admin/Server/Services/SupportService.cs

```diff
@@ -160,8 +160,15 @@ private async Task<PatientSupportDetails> GetPatientSupportDetailsAsync(UserProf
                 ? new PatientDetailsQuery(Hdid: queryString)
                 : new PatientDetailsQuery(Phn: queryString);
 
-            PatientQueryResult result = await this.patientRepository.Query(query, ct).ConfigureAwait(true);
-            return result.Items.SingleOrDefault();
+            try
+            {
+                PatientQueryResult result = await this.patientRepository.Query(query, ct).ConfigureAwait(true);
+                return result.Items.SingleOrDefault();
+            }
+            catch (ProblemDetailsException)
```

Comment:
You may want to be more specific here eventually

Reply:
Do we not have a value that can determine if its a "not found" error vs others?

Reply:
Fixed.